components/HomeHeader.vue

```diff
@@ -64,10 +65,8 @@
                 tracks
                 runtime
                 rating
-                class="mt-2"
+                class="my-2"
               />
-              <!-- eslint-disable-next-line vue/no-v-html -->
-              <p class="mt-2" v-html="getOverview(item)" />
```

Comment:
Why remove `getOverview`?

Reply:
I mentioned it on Matrix, but should've made it part of the PR as well:

We can't really control the length of the overview and it's very variable. As a result, it can be a sentence long or multiple paragraphs.  
In addition, there's no good way to limit the length of the test: we either cut between sentences or between paragraphs or in the middle of a sentence. None of these are really good solutions.  
We also can only really show the overview at 1920x1080 or higher, which is a bit annoying.

So it was removed to make the header a bit nicer, less complex and not have to resort to a dirty way of clamping lines of text. The overview is available on the details page anyway, so it shouldn't be an issue.

Reply:
Ok, that makes sense, not much we can do about it.